edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest/FakeModuleManager.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Agent.Core;
+    using Microsoft.Azure.Devices.Edge.Agent.Edgelet;
+    using Microsoft.Azure.Devices.Edge.Agent.Edgelet.Models;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public class FakeModuleManager : IModuleManager
```

Comment:
It is not fake, just an implementation for Kubernetes.
```suggestion
    public class KubernetesModuleManager : IModuleManager
```

Reply:
It's a lightweight, test-only implementation so it's appropriate to use a test-specific term like "fake" (see Meszaros' definitions of [test doubles](http://xunitpatterns.com/Test%20Double.html)). That said, it looks like this object is only injected into `KubernetesRuntimeInfoProvider` to fulfill a dependency--it doesn't actually do anything--so technically "dummy" is more appropriate. However there's some overlap in terminology, so it's probably not worth getting pedantic.  ;)

Reply:
Let's make it dummy!